app-text/multimarkdown/multimarkdown-6.4.0.recipe

```diff
@@ -0,0 +1,66 @@
+SUMMARY="Lightweight markup processor to produce HTML, LaTeX, and more"
+DESCRIPTION="MultiMarkdown, or MMD, is a tool to help turn minimally marked-up \
+plain text into well formatted documents, including HTML, PDF (by way of LaTeX), \
+OPML, or OpenDocument (specifically, Flat OpenDocument or ‘.fodt’, which can \
+in turn be converted into RTF, Microsoft Word, or virtually any other \
+word-processing format)."
+HOMEPAGE="https://fletcherpenney.net/multimarkdown/"
+COPYRIGHT="2016-2018, Fletcher T. Penney"
```

Comment:
```suggestion
COPYRIGHT="2016-2018 Fletcher T. Penney"
```

Reply:
From what I can see, [these][0] libraries are built directly into `multimarkdown`, as such, this package is subjected to their copyrights as well.

[0]: https://github.com/fletcher/MultiMarkdown-6/blob/57d640bc924232f9f94df3d6100ea37c4cc3fac4/LICENSE.txt#L13-L29